### PR TITLE
Removing outdated VS references (Part 1)

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #1795: This is Part 1 of the updates, so let's call it :shark: *Enough Said!* :shark:

* Addresses token-based references to prior VS versions.
* Removes and redirects the breaking changes doc for VS2013

### CODE REVIEW

File: docs/csharp/programming-guide/interop/walkthrough-office-programming.md [hunk 1]
```diff
@@ -36,16 +36,14 @@ translation.priority.ht:
...
   
- This walkthrough demonstrates the new features in the context of Office programming, but many of them are also useful in general programming. In the walkthrough, you will first use an Excel Add-in application to create an Excel workbook. You will then create a Word document that contains a link to the workbook. Finally, you will see how the PIA dependency can be turned on and off.  
+This walkthrough demonstrates these features in the context of Office programming, but many of these features are also useful in general programming. In the walkthrough, you use an Excel Add-in application to create an Excel workbook. Next, you create a Word document that contains a link to the workbook. Finally, you see how to enable and disable the PIA dependency.  
   
 ## Prerequisites  
- You must have Microsoft Office Excel 2013 (or version 2007 or later) and Microsoft Office Word 2013 (or version 2007 or later) installed on your computer to complete this walkthrough.  
+You must have Microsoft Office Excel 2013 (or version 2007 or later) and Microsoft Office Word 2013 (or version 2007 or later) installed on your computer to complete this walkthrough.
```
Comment (mairaw) [inline docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: these requirements are kind of odd. We should revisit this when splitting the topic for VB and C#

**Review by mairaw — APPROVED** [review]
LGTM! @BillWagner do you wanna take a look?

File: docs/csharp/programming-guide/interop/walkthrough-office-programming.md (hunk 1 above)
Comment (GuardRex) [inline docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: Probably should just say "Microsoft Office Excel 2007 or later" and "Microsoft Office Word 2007 or later" or just go with "Microsoft Office Excel" and "Microsoft Office Word" at this point, given that 2007 was 10 years ago.

Comment (mairaw) [inline docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: I agree.

Comment (GuardRex) [inline docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: There's a whole lot of "clicking" in this. I'll come back on a future PR and tidy this doc up. For now, I fixed that spot :point_up:.

**Pull request merged**